### DISCUSSION TITLE
GetTableList Funktion of Table

### DISCUSSION THREAD

**Discussion by boyi01** [JSQLParser/JSqlParser#1188] [Q&A]
Hey,

i want to  analyze sqls regarding from which table there reading and to which table there writing (changing stuff).

is there any possiblity to geht these information with JSqlParser?

**Comment by manticore-projects** [comment]
Greetings.

Yes, this can be achieved with JSQLParser:

1) Tables to be written to would be found in `INSERT`, `MERGE`, `UPDATE`, `DELETE` and `TRUNCATE TABLE` statements
2) Tables to be read from would be found in `SELECT` statements and `JOIN` fragments

Best regards
Andreas
